src/qtcutil.h

```diff
@@ -21,6 +21,8 @@
 #ifndef QTCUTIL_H
 #define QTCUTIL_H
 
+#define QTC_CALL_SIZE 15
```

Comment:
This macro is already defined in [qtcvars.h](https://github.com/Tlf/tlf/blob/988dad277c78e14f096b86d4c23db41eb470f363/src/qtcvars.h#L39). You can consider to use that